assets/javascripts/kitten/components/form/text-input.test.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { expect } from 'chai'
+import { shallow } from 'enzyme'
+import { TextInput } from 'kitten/components/form/text-input'
+
+describe('Text-input with default props', () => {
+  const defaultComponent = shallow(
+  <TextInput />)
+
+  it('has a default text-input attributes', () => {
+    expect(defaultComponent.find('.k-TextInput')).to.have.length(1)
+  })
+
+  it('has a default tag', () => {
+    expect(defaultComponent).to.have.tagName('input')
+  })
+
+  it('has a valid class', () => {
+    expect(defaultComponent.find('.is-valid')).to.have.length(0)
+  })
+
+  it('has a error class', () => {
+    expect(defaultComponent.find('.is-error')).to.have.length(0)
+  })
+
+  it('has a tiny class', () => {
```

Comment:
Ces tests qui testent qu'il n'y a PAS cet élément devraient plutôt dire `does not contain a is-**** element`, non ?